spec/helpers.lua

```diff
@@ -1586,10 +1586,13 @@ end
 -- NOTE: this function is not available for DBless-mode
 -- @function wait_for_all_config_update
 -- @tparam[opt=30] number timeout maximum time to wait
--- @tparam[opt] number admin_client_timeout, to override the default timeout setting
+-- @tparam[opt] number admin_client_timeout, to override the default timeout setting of admin client
 -- @tparam[opt] number forced_admin_port to override the default port of admin API
+-- @tparam[opt] number proxy_client_timeout, to override the default timeout setting of proxy client
+-- @tparam[opt] number forced_proxy_port to override the default port of proxy client
 -- @usage helpers.wait_for_all_config_update()
-local function wait_for_all_config_update(timeout, admin_client_timeout, forced_admin_port)
+local function wait_for_all_config_update(timeout, admin_client_timeout, forced_admin_port,
```

Comment:
not a blocker but for future us: with the argument list becoming long and user might not understand what each one means from its type, it will make more sense to make the function accept a table, so you can use key/value to make arguments more readable. (exceptions apply for performance critical path.)

Reply:
done